apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/rivers/StartInformasjonsbrevgenereringRiver.kt

```diff
@@ -0,0 +1,122 @@
+package no.nav.etterlatte.rivers
+
+import kotliquery.TransactionalSession
+import no.nav.etterlatte.brev.brevbaker.EtterlatteBrevKode
+import no.nav.etterlatte.funksjonsbrytere.FeatureToggle
+import no.nav.etterlatte.funksjonsbrytere.FeatureToggleService
+import no.nav.etterlatte.libs.common.behandling.SakType
+import no.nav.etterlatte.libs.common.event.BrevEventKeys
+import no.nav.etterlatte.libs.common.rapidsandrivers.EVENT_NAME_KEY
+import no.nav.etterlatte.libs.common.rapidsandrivers.SAK_TYPE_KEY
+import no.nav.etterlatte.libs.database.Transactions
+import no.nav.etterlatte.libs.database.hentListe
+import no.nav.etterlatte.libs.database.oppdater
+import no.nav.etterlatte.libs.database.transaction
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.RapidsConnection
+import org.slf4j.LoggerFactory
+import rapidsandrivers.FNR_KEY
+import java.util.UUID
+import javax.sql.DataSource
+import kotlin.concurrent.thread
+
+class StartInformasjonsbrevgenereringRiver(
+    private val repository: StartBrevgenereringRepository,
+    private val rapidsConnection: RapidsConnection,
+    private val featureToggleService: FeatureToggleService,
+    private val sleep: (millis: Long) -> Unit = { Thread.sleep(it) },
+    private val iTraad: (handling: (() -> Unit)) -> Unit = { thread { it() } },
+) {
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    init {
+        startBrevgenerering()
+    }
+
+    private fun startBrevgenerering() {
+        logger.info("Starter å generere informasjonsbrev for mottakerne som ligger i databasetabellen")
+        val fnr =
```

Comment:
Å bruke fnr her er ei veldig generaliserande tilnærming for informasjonsbrev. I informasjonsbreva vi treng dette for no (du får utsatt klagefrist fordi du får brevet seint) har vi dei relevante behandling-id-ane allereie, så vi kan heller leggje inn dei. Men tenkte litt framover da eg laga denne. Usikker her. Eventuelt om vi lagar denne databasetabellen så han kan ta behandlingid og/eller fnr (to separate felt), og så bruker vi det som er definert 🤔

Reply:
Endte opp no med å skrive om så vi her kan leggje inn fnr eller behandlingid, og så skal resten av flyten takle begge delar